src/router/index.js

```diff
@@ -129,8 +129,9 @@ const router = new VueRouter({
 });
 
 router.beforeEach((to, from, next) => {
-  if (tokenService.getPrivilegeLevel() < 0) {
-    if (to.name === 'Login' || to.name === 'SignUp' || to.path === '/forgot-password-reset/*') next();
+  if (tokenService.getPrivilegeLevel() === privilegeConstants.NONE) {
+    if (to.path === '/forgot-password-reset/*') next();
+    else if (to.name === 'Login' || to.name === 'SignUp') next();
```

Comment:
Does this work if you leave them as one `if` statement and move `to.path === '/forgot-password-reset/*'` to be the first check instead of the third one?